### DISCUSSION TITLE
How to use light_pollution keyword in landscape.ini?

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3174] [General]
Hi guys, 

I read this discussion: https://github.com/Stellarium/stellarium/issues/1586

As far as I understand it, it was planned to change the unit of the "light_pollution" keyword in the landscape.ini file from bortle class to a more scientific unit. But as far as I can tell, there is no final conclusion about that. 

So what is the current way to specify the light pollution for a location using the landscape.ini file? And do I have to enable the usage of this keyword somewhere in the GUI or is it used by default? I recently tried entering bortle class numbers for this keyword, but they have no effect. 

Kind regards
Gerrit

**Comment by 10110111** [comment]
Use `light_pollution_luminance` to specify your value in cd/m<sup>2</sup>. The old keyword should then be removed.

> I recently tried entering bortle class numbers for this keyword, but they have no effect.

They should have an effect if the new keyword is missing.

**Reply by gzotti** [reply]
We have forgotten to add this info to the User Guide, sorry.
@10110111 Do you have a table Bortle vs. cd/m² somewhere? The User Guide lists yet different units in Appendix B.

**Reply by 10110111** [reply]
> Do you have a table Bortle vs. cd/m² somewhere?

No, this can only be computed indirectly via `StelCore::nelmToLuminance` and `StelCore::bortleScaleIndexToNELM`. Also not to forget that each Bortle scale index belongs to a range of luminances/NELMs.

**Reply by gzotti** [reply]
Sure, it's a quality range, not a precise value. Still, an area diagram of cd/m² values can show 9 distinctly colored areas called "Bortle zone".

**Reply by GerritErdt** [reply]
Thanks for the fast reply, that solved the isse. I don't really know what I did wrong earlier, but now everything works as expected!

Wouldn't it be much more practical to add a keyword to input the sky brightness in mag/arcsec^2 ? That is a much more convenient number than cd/m^2 for most astronomers.

**Reply by 10110111** [reply]
It might be more practical, but cd/m<sup>2</sup> was chosen because it's the standard physical unit, and it's also used in the computations internally. Maybe we could add another keyword to specify in another units, but for now you can use Stellarium's light pollution GUI to convert between units: *View*→*Sky*→*Light pollution*→*manual from SQM*, choose your input unit, enter the value, choose target unit, get the new value.

**Comment by GerritErdt** [comment]
Alright, thanks for the explanation, that makes sense!

BTW, the support here is awesome!
